editors/code/src/ctx.ts

```diff
@@ -103,6 +105,10 @@ export class Ctx implements RustAnalyzerExtensionApi {
     ) {
         extCtx.subscriptions.push(this);
         this.statusBar = vscode.window.createStatusBarItem(vscode.StatusBarAlignment.Left);
+        this.testController = vscode.tests.createTestController(
```

Comment:
Maybe add an option to control whether the feature is enabled? The default might be `true`(although it's kind of aggressive for me, because the API are all experimental), but an option is a good escape hatch if there is something wrong.

Reply:
Agreed, config (defaulted to false) and a capability is the least we should do here. https://github.com/rust-lang/rust-analyzer/pull/16773

Reply:
Why should the default be false? Does the presence of this feature interfere with normal operation?

Reply:
This is hot of the press, it might have issues we don't know about and we don't know the perf impact yet. Turning it off by default (and then stating that in the changelog how to enable) seems like the better starting experience, I reckon (at least judging from the issue) that enough people are interested in this and enable to it to give us relevant testing feedback.